politeiawww/README.md

```diff
@@ -71,13 +69,26 @@ password of your root user. The politeiawwwpass is the password that will be
 set for the politeiawww user when it's created.
 
 ```
-$ cd $GOPATH/src/github.com/decred/politeia/politeiawww/scripts/userdb
+$ cd $GOPATH/src/github.com/decred/politeia/scripts/userdb
 $ env \
   MYSQL_ROOT_PASSWORD=rootpass \
   MYSQL_POLITEIAWWW_PASSWORD=politeiawwwpass \
   ./mysqlsetup.sh
 ```
 
+You will need to use the `--userdb=mysql` flag when starting politeiawww or 
+add `userdb=mysql` to the `politeiawww.config` file that is setup in the steps
+below.
+
+Also, an encryption key is required when using a MySQL database, use
+`politeiawww_dbutil` cmd tool to create one: 
+```
+politeiawww_dbutil -createkey 
+```
+Then, add  `encryptionkey` to `politeiawww.config` or use 
```

Comment:
The encryption key path defaults now on politeiawww startup, right?  So no need to add this to the config file.

Reply:
I would just take this paragraph out to simplify it.

Reply:
true